src/OutputCacheProvider/Properties/AssemblyInfo.cs

```diff
@@ -14,7 +14,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("OutputCacheProvider")]
-[assembly: AssemblyCopyright("Copyright © 2014")]
+[assembly: AssemblyCopyright("Copyright © 2022")]
```

Comment:
Let's pull out the updates to other projects into a separate PR (date update, string interpolation, etc.). That way we'll have a clear PR that only introduces the new output cache provider middleware